docs/source/bibliography.bib

```diff
@@ -89,4 +89,19 @@ @article{maheshwari-et-al:noninvasive:2018
   volume        = {127},
   year          = {2018},
   bdsk-url-1    = {https://journals.lww.com/anesthesia-analgesia/fulltext/2018/08000/a_randomized_trial_of_continuous_noninvasive_blood.21.aspx}
-}
\ No newline at end of file
+}
+
+@article{10.1016/0013-46947090143-4,
+  title = {{{EEG}} Analysis Based on Time Domain Properties},
+  author = {Hjorth, Bo},
+  year = {1970},
+  month = sep,
+  journal = {Electroencephalography and Clinical Neurophysiology},
+  volume = {29},
+  number = {3},
+  pages = {306--310},
+  issn = {0013-4694},
+  doi = {10.1016/0013-4694(70)90143-4},
+  urldate = {2025-06-27},
+  abstract = {A method to describe the general characteristics of an EEG trace in a few quantitative terms is introduced. Its descriptive parameters are entirely based on time, but they can be derived also from the statistical moments of the power spectrum. Thus the method provides a bridge between a physical time domain interpretation and the conventional frequency domain description. Further, the parameters are based on the concept of variance, giving them an additive property so that the measured values pertain also to any basic elements from which a complex curve may be composed by superposition. The proposed method offers a way to on-line measurement of basic signal properties by means of a time-based calculation, requiring less complex equipment compared to conventional frequency analysis. The data-reducing capability of the parameters has been experimentally stated in the recording of ``sleep profiles''. R{\'e}sum{\'e} L'auteur introduit une m{\'e}thode de description des caract{\'e}ristiques g{\'e}n{\'e}rales d'un trac{\'e} EEG en un nombre limit{\'e} de termes quantitatifs. Ses param{\`e}tres descriptifs sont enti{\`e}rement bas{\'e}s sur le temps, mais peuvent {\^e}tre d{\'e}riv{\'e}s {\'e}galement des moments statistiques du spectre de puissance. Ainsi, cette m{\'e}thode fait la jonction entre une interpr{\'e}tation du domaine des s{\'e}ries temporelles physiques et la description du domaine fr{\'e}quentiel conventionnel. De plus les param{\`e}tres sont bas{\'e}s sur le concept de variance, leur donnant une propri{\'e}t{\'e} suppl{\'e}mentaire de telle sorte que les valeurs mesur{\'e}es se rapportent {\'e}galement {\`a} chaque {\'e}l{\'e}ment de base {\`a} partir duquel une courbe complexe peut {\^e}tre compos{\'e}e par superposition. La m{\'e}thode propos{\'e}e offre un moyen de mesurer ``on-line'' des propri{\'e}t{\'e}s de base du signal au moyen d'un calcul bas{\'e} sur le temps, n{\'e}cessitant un {\'e}quipement moins complexe que l'analyse de fr{\'e}quence conventionnelle. La capacit{\'e} de r{\'e}duction des donn{\'e}es des param{\`e}tries a {\'e}t{\'e} sp{\'e}cifi{\'e}e exp{\'e}rimentalement dans l'enregistrement des ``profils de sommeil''.}
```

Comment:
I'd prefer (although this is not a strong preference) removing the abstract from our local .bib file.

```suggestion
```